shared/search/helpers.js

```diff
@@ -33,7 +33,9 @@ type OutProps = {
 }
 */
 const clearSearchHoc = withHandlers({
-  onClearSearch: ({onExitSearch}) => () => onExitSearch(),
+  // use existing onClearSearch if exists. TODO change how this whole thing works. so confusing
```

Comment:
how search works is very convoluted. i think we can do something simpler but lets leave that for later. for now if you pass in an onClearSearch it won't blow it away